newIDE/app/src/EventsFunctionsExtensionEditor/EventsFunctionConfigurationEditor/EventsFunctionParametersEditor.js

```diff
@@ -462,6 +469,10 @@ export default class EventsFunctionParametersEditor extends React.Component<
                                   value="objectAnimationName"
                                   primaryText={t`Object animation (text)`}
                                 />
+                                <SelectOption
+                                  value="uid"
+                                  primaryText={t`Unique event ID (text)`}
```

Comment:
Is this an event unique id, a condition unique id, a parameter unique id?

Reply:
Well it technically is meant to be a condition/action/expression unique ID, but the GDevelop terminology doesn't have a word for that.

Reply:
`Unique ID` or `UUID` have more sense and are common, although misleading with potential `UUID` from an instances.
 All people talking about UUID make reference to UUID from C2/3, for not losing everyone it's fine to differentiate things, although technically all the words are valid.

 I would prefer `Generated ID`

Reply:
No, an identifier is not unique by definition, multiple things can be identified by an identifier, and an identifier can be duplicated/reused unless it is unique. [See the distinction between identifier and unique identifier on wikipedia](https://en.wikipedia.org/wiki/Identifier). In my opinion, `Generated identifier` is too vague (when is it generated, is it unique, what does it identify) and doesn't tell anything about what it is/what it does/what it is used for. It's yet another piece of terminology to learn. Something like "Unique event identifier" or "Unique instruction identifier" tells you directly "Aha, this identifies the event/instruction it is used in!".